docs/csdk-guide.md

```diff
@@ -214,6 +226,122 @@ release the memory and destroy JVM.
     void withHadoopConf(char *key, char *value);
 ```
 
+<<<<<<< HEAD
```

Comment:
what's this?

problem while resolving a conflict? please check and redo this file.

Reply:
sorry, rebase error, rebase many times before